js/init.js

```diff
@@ -12,34 +13,44 @@
                 running = false;
             });
         },
+
         readyFn = function() {
+
             var isTouch = window.hasOwnProperty('ontouchstart') && window.ontouchstart,
                 Elements = {
-                    'sticky': document.querySelector('.sticky'),
+                    'header': document.querySelector('header'),
                     'body': document.querySelector('body'),
                     'title': document.querySelector('.page-title'),
                     'trigger': document.querySelector('.trigger'),
+                    'pageLinks': document.querySelector('.page-links'),
+                    'social': document.querySelector('.social'),
                     'menu': document.querySelector('.mobile-menu'),
                     'authors': document.querySelector('main section .authors'),
                 },
                 Positions = {
-                    'sticky': Elements.sticky.offsetTop,
+                    'pageLinks': Elements.pageLinks.offsetTop,
                     'doc': window.scrollY,
                 },
                 Handler = isTouch ? 'touchstart' : 'click';
 
+            if (!~Elements.header.classList.value.indexOf('hasImage')) {
```

Comment:
Happy with this being one from a selection of images each time the page is loaded rather than each time it is rendered? (ie. done in JS vs being done in Jekyll?)